library/Debug.cpp

```diff
@@ -80,11 +80,11 @@ static color_value selectColor(const DebugCategory::level msgLevel)
 {
     switch(msgLevel) {
     case DebugCategory::LTRACE:
-        return COLOR_GREY;
+        return COLOR_BROWN;
     case DebugCategory::LDEBUG:
         return COLOR_LIGHTBLUE;
     case DebugCategory::LINFO:
-        return COLOR_CYAN;
+        return COLOR_GREY;
```

Comment:
I use a light terminal theme (weird, I know), so I would somewhat prefer COLOR_RESET:

![image](https://user-images.githubusercontent.com/3719547/160753007-b5567d9d-7bf6-4e92-9d7c-21a338d22c79.png)

over COLOR_GREY:

![image](https://user-images.githubusercontent.com/3719547/160752980-d07de521-45d8-4156-90b0-02b3398db2ac.png)

...although I can still read either. I'd definitely like to change the COLOR_WHITE below to COLOR_RESET, for obvious reasons.

Maybe "INFO" should just fall through to the default color. Seems like a pretty "normal" type of message in general, to me.

Reply:
done